packages/gatsby/lib/bootstrap/index.js

```diff
@@ -305,8 +308,8 @@ module.exports = async (program: any) => {
 
   console.log(`created js pages`)
 
-  return new Promise(resolve => {
-    queryRunner(thing => {
+  const finishPromise = new Promise(resolve => {
```

Comment:
Oh oops that was leftover from when I was experimenting with something. Fixed to just returning the promise.